### PR TITLE
Update toggle MDL2 styling

### PR DESCRIPTION
**Pull request by philworthington**
Update toggle with correct border-color on hover and focus. Add border-color to transition-property.

### CODE REVIEW

File: src/components/Toggle/Toggle.scss [hunk 1]
```diff
@@ -211,9 +212,10 @@
   border: 2px solid $ms-color-neutralTertiary;
   border-radius: 20px;
   cursor: pointer;
-  transition-property: background, left;
+  transition-property: background, left, border-color;
   transition-duration: 250ms;
   transition-timing-function: cubic-bezier(.4, 0, .23, 1);
+  outline: 0;
```
Comment (Jahnp) [inline src/components/Toggle/Toggle.scss]: Is this ok for high contrast mode? Are the border styles here enough?

**Comment by Jahnp** [comment]
Approved

**Comment by mikewheaton** [comment]
Approved 👍

File: src/components/Toggle/Toggle.scss (hunk 1 above)
Comment (philworthington) [inline src/components/Toggle/Toggle.scss]: The outline was going around the label area of the toggle which would not be the appropriate high contrast style. We might have to do another high contrast pass of all of the components after the mdl2 updates have been made.

**Pull request merged**